app/adapters/alma_adapter/marc_record.rb

```diff
@@ -0,0 +1,69 @@
+class AlmaAdapter
+  class MarcRecord < SimpleDelegator
```

Comment:
Add a class level comment just describing what this class is / does? Seems like a wrapper class.